src/sign.py

```diff
@@ -103,6 +103,21 @@ def _arguments() -> argparse.Namespace:
         default=True,
         dest="use_ambient_credentials",
     )
+    sigstore.add_argument(
+        "--staging",
+        help="Use Sigstore's staging instances, instead of the default"
+        + " production instances",
```

Comment:
Let's use implicit string concatenation instead of `+`. See https://github.com/sigstore/model-transparency/blob/a0fa17360c9d768ea532404eaaf6ef1fb54627dc/src/model_signing/serialization/serialize_by_file.py#L60-L63 for an example.

Reply:
Done. Was copying style from the argument above.